packages/@o3r/schematics/src/tasks/ng-add/index.ts

```diff
@@ -31,6 +31,7 @@ export interface NgAddPackageOptions {
   skipNgAddSchematicRun?: boolean;
 }
 
+/** @deprecated use {@link setupDependencies} instead, will be removed in V11 */
```

Comment:
v12?

Reply:
nop, will not wait 1 year to remove a schematic helper